packages/assets-controllers/src/NftController.test.ts

```diff
@@ -1437,48 +1472,57 @@ describe('NftController', () => {
           name: 'Kudos Name (directly from tokenURI)',
           description: 'Kudos Description (directly from tokenURI)',
         });
-      const { selectedAddress, chainId } = nftController.config;
-      sinon
-        // TODO: Replace `any` with type
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        .stub(nftController, 'getNftInformationFromApi' as any)
-        .returns(undefined);
+
+      nock('https://nft.api.cx.metamask.io')
+        .get(
+          '/tokens?chainIds=1&tokens=0x2aEa4Add166EBf38b63d09a75dE1a7b94Aa24163%3A1203&includeTopBid=true&includeAttributes=true&includeLastSale=true',
+        )
+        .reply(404, { error: 'Not found' });
```

Comment:
nit: Would it make sense to convert both the url and the query string into template strings and extract constants? [This](https://github.com/MetaMask/core/pull/4310/files#diff-22f99da950cfe9d51506651602f04e50afcd3805291cbfa9e18165dce3568f25R1795) and [this](https://github.com/MetaMask/core/pull/4310/files#diff-22f99da950cfe9d51506651602f04e50afcd3805291cbfa9e18165dce3568f25R1937) seem a bit more readable and reusable.